app/src/main/java/com/example/memeitupapp/ui/gridmemes/GridMemesFragment.kt

```diff
@@ -23,20 +26,44 @@ class GridMemesFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        gridMemesViewModel = ViewModelProvider(this).get(GridMemesViewModel::class.java)
```

Comment:
you don't need a ViewModelProvider here.

viewmodel = viewModel(Model(MemeRepo()))